<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CaptureWindow</class>
 <widget class="QMainWindow" name="CaptureWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>944</width>
    <height>726</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="exposureBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>10</y>
      <width>261</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Exposure</string>
    </property>
    <widget class="QComboBox" name="binningComboBox">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>70</y>
       <width>111</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="binningLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>62</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Binning:</string>
     </property>
    </widget>
    <widget class="QLabel" name="timeLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>62</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Time:</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="timeText">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>104</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="coolerBox">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>230</y>
      <width>261</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Cooler</string>
    </property>
   </widget>
   <widget class="QPushButton" name="captureButton">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>590</y>
      <width>114</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Capture</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>380</y>
      <width>261</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Correction</string>
    </property>
   </widget>
   <widget class="QLabel" name="imageLabel">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>15</y>
      <width>631</width>
      <height>671</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>944</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>captureButton</sender>
   <signal>clicked()</signal>
   <receiver>CaptureWindow</receiver>
   <slot>startCapture()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>790</x>
     <y>638</y>
    </hint>
    <hint type="destinationlabel">
     <x>693</x>
     <y>672</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>startCapture()</slot>
 </slots>
</ui>
